hoodie-client/src/main/java/com/uber/hoodie/io/compact/strategy/DayBasedCompactionStrategy.java

```diff
@@ -32,7 +32,7 @@
  * This strategy orders compactions in reverse order of creation of Hive Partitions. It helps to
  * compact data in latest partitions first and then older capped at the Total_IO allowed.
  */
-public class DayBasedCompactionStrategy extends BoundedIOCompactionStrategy {
+public class DayBasedCompactionStrategy extends UnBoundedCompactionStrategy {
```

Comment:
Can we extend directly from CompactionStrategy as UnBoundedCompactionStrategy does not seem to add any functionality